addons/website_event_sale/controllers/main.py

```diff
@@ -58,16 +58,11 @@ def registration_confirm(self, event, **post):
 
         return request.redirect("/shop/checkout")
 
-    def _add_event(self, event_name="New Event", context={}, **kwargs):
-        try:
-            res_id = request.env.ref('event_sale.product_product_event').id
-            context['default_event_ticket_ids'] = [[0, 0, {
-                'name': _('Subscription'),
-                'product_id': res_id,
-                'deadline': False,
-                'seats_max': 1000,
-                'price': 0,
-            }]]
-        except ValueError:
-            pass
-        return super(website_event, self)._add_event(event_name, context, **kwargs)
+    def _add_event(self, event_name="New Event"):
+        event = super(website_event, self)._add_event(event_name)
+
+        event.event_ticket_ids.write({
+            'deadline': False,
+            'seats_max': 1000,
+        })
+        return event
```

Comment:
Why write ? why not setting default value, I know there is default function on field event_ticket_ids, but now with this change you are doing default function call + write call